### PR TITLE
Language and form suggestions

### PR DESCRIPTION
**Pull request by Cortey**
**Description**
Changed the .md files according to Technical Writer's suggestions.

**Changes**
- Replaced the diagram file with updated version.
- Updated documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 ## Overview
 
-Kyma Showcase is a project about object recognition relying on GCP API and Serverless Functions Workflow. A customer uploads an image on the front page. Then the image is displayed in the feed. Clicking on any image in the feed redirects you to its details page showing all the information acquired from processing that image. <!--maybe we could add some information examples>
+Kyma Showcase is a project about object recognition relying on GCP API and Serverless Functions Workflow. A customer uploads an image on the front page. Then the image is displayed in the feed. Clicking on any image in the feed redirects you to its details page showing all the information acquired from processing that image such as prevalent mood, labels describing the image or objects present in the image. More information about Kyma Showcase can be found in [Details](./docs/details.md) and [Architecture](./docs/architecture.md).
```
Comment (mmitoraj) [inline README.md:8]:
```suggestion
Kyma Showcase is a project about object recognition relying on GCP API and Serverless Functions Workflow. A customer uploads an image on the front page. Then the image is displayed in the feed. Clicking on any image in the feed redirects you to its details page showing all the information acquired from processing that image such as prevalent mood, labels describing the image itself, or objects in it. See the [Details](./docs/details.md) and [Architecture](./docs/architecture.md) sections for more information about Kyma Showcase.
```

File: docs/architecture.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
 1. Frontend gets all images stored in the Redis Database and displays them in the feed.
 2. A user uploads an image using Frontend's UI.
-3. The image is stored in the database and then displayed in the feed. <!--the step is similar to the first one>
-4. Backend sends an event containing the database ID used to access the image.
-5. Serverless Functions are triggered by the event and get the image from the database using the given ID.
+3. The newly uploaded image is stored in the Redis Database and then displayed in the feed.
+4. Backend sends an event containing the Redis Database ID used to access the image.
+5. Serverless Functions are triggered by the event and get the image from the Redis Database using the given ID.
 6. The Functions send the image in base64 to Google Cloud Platform for processing.
```
Comment (mmitoraj) [inline docs/architecture.md:12]:
```suggestion
6. The Functions send the image in base64 to the Google Cloud Platform (GCP) for processing.
```

This is the first time you use "Google Cloud Platform". Let's add the abbreviation in brackets here and use GCP from now on in the doc.

File: docs/architecture.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-7. The database entry is updated with newly obtained image details.
+7. The Redis Database entry is updated with newly obtained image details.
 8. Depending on the details content, a new event may be sent triggering next Functions.
-9. Upon clicking a single image in the Frontend UI, a details page is displayed listing all the information recieved from the Google Cloud Platform.
+9. Upon clicking a single image in the Frontend UI, a details page is displayed listing all the information recieved from the Google Cloud Platform(GCP).
```
Comment (mmitoraj) [inline docs/architecture.md:15]:
```suggestion
9. Upon clicking a single image in the Frontend UI, a details page is displayed listing all the information received from the GCP.
```

**Review by mmitoraj — CHANGES_REQUESTED** [review]
The numbers on the diagram look good :) Just a few small comments.

**Review by mwisnia97 — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request closed without merging**